config/policies.js

```diff
@@ -75,8 +75,7 @@ module.exports.policies = {
   /////////////////////////////////
   // DEVELOPMENT Or Staging ONLY //
   /////////////////////////////////
-  'game/stack-deck': 'developmentOrStagingOnly',
-  'game/clear-game': 'developmentOrStagingOnly',
+
```

Comment:
We should also delete the policy for the load-fixture action below as its now covered by the TestController policy